README.md

```diff
@@ -171,11 +171,13 @@ steps:
     cache: true
     cache-key: 'flutter-:os:-:channel:-:version:-:arch:-:hash:' # optional, change this to force refresh cache
     cache-path: '${{ runner.tool_cache }}/flutter/:channel:-:version:-:arch:' # optional, change this to specify the cache path
+    pub-cache-key: 'flutter-pub:os:-:channel:-:version:-:arch:-:hash:' # optional, change this to force refresh cache of dart pub get dependencies
+    cache-path: '${{ runner.tool_cache }}/flutter/:channel:-:version:-:arch:' # optional, change this to specify the cache path
```

Comment:
Hi @jorgenpt! Nice PR, I was facing the same issue.
double cache-path here, line 173 == line 175

Reply:
Also, let's please remove `-:hash:` from line cache-key (line 172)

Reply:
Ah, thanks for pointing out the mistake in the example!